x-pack/plugins/synthetics/common/runtime_types/dynamic_settings.ts

```diff
@@ -38,6 +38,27 @@ export const DynamicSettingsSaveType = t.intersection([
   }),
 ]);
 
+export const LocationMonitorsType = t.intersection([
+  t.partial({
+    payload: t.type({
+      aggregations: t.type({
+        locations: t.type({
+          buckets: t.array(
+            t.type({
+              key: t.string,
+              doc_count: t.number,
+            })
+          ),
+        }),
+      }),
+    }),
+  }),
+  t.type({
+    status: t.number,
+  }),
+]);
```

Comment:
i think this type should be bit more simple. We should do processing on server side and return the processed results to client.

Reply:
++ that's fine, I'll move the complexity